worker/worker/transcode.go

```diff
@@ -194,3 +194,24 @@ func reportProgress(stream *StreamContext, p chan int32) error {
 		}
 	}
 }
+
+func transcodeAudio(ctx *StreamContext) error {
+	S.startTranscodingAudio(ctx.getStreamName())
```

Comment:
I'd love to hear you opinions on starting and ending (with defer) inside the "working" instead of inside the handler function. Any drawbacks?

Reply:
Inside the "working" function would actually be the correct way. Currently, the status is only cleared after all jobs are done.

Unrelated; the workload checks are a little overengineered imo, we could also just add 1 for every stream, and it's post-processing steps the worker is handling

Reply:
Agree. I think we talked about this last week, the worker probably needs a bit of refactoring 😬

Reply:
I think  I'll try to rework the worker